Infection report

mapesa opened the pull request:
@kitsao , please review the infection report.
- I'm using a RAW query
- I think it'll have to do for now. Other fancy tweaks can be done when we have less work pressure.

kitsao commented:
@mapesa , here we go...
Generally, the report is good, work has been done! 
Time taken to load data for one year is short enough, just like we wanted.
I also agree that the raw query will do for the moment. A few things I have identified...
- Some test types like Borrelia doesn't have rows for 'Male' gender for both results. Maybe it's by design, not sure though.
- I noticed that tests that should have counts>0 for one gender such as pregnancy test have counts>0 for males too. I checked with the current BLIS and it seems to be a flaw with the data.
- Counts may not tally with the counts in the current BLIS but I guess it's as a result of the age ranges defined in either.
- There is an incomplete table row at the end of the table which seems to be part of the 'Positive' results for 'Male' gender, VDRL.

Let's go through the above, agree on them and I think we'll be good to merge.

mapesa commented:
@kitsao,
- I've fixed the issue of not showing the gender - its now shown for all tests
- I've fixed the table ending.
- May be we can leave **minor** discrepancies in the counts to the verification process (as long as the report logic is ok)

kitsao commented:
@mapesa , this thing is good to go!
